From 807d70cd1ed095010818ebd268db7bc45df5233c Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Wed, 30 Aug 2017 14:54:18 +0300
Subject: [PATCH 2117/2241] Revert "ARM: mvebu: coherency_available based on
 current coherency_type"

This reverts commit 9f8a6a059dc259825f587ddea64bb4a9339f2fb1.

Change-Id: I31d80f9a2bd0b4820d801ea78f3c7bea13e2970d
Reviewed-on: http://vgitil04.il.marvell.com:8080/43558
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
Tested-by: Neta Zur Hershkovits <neta@marvell.com>
---
 arch/arm/mach-mvebu/coherency.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mvebu/coherency.c b/arch/arm/mach-mvebu/coherency.c
index 7f03115..3285d0e 100644
--- a/arch/arm/mach-mvebu/coherency.c
+++ b/arch/arm/mach-mvebu/coherency.c
@@ -321,7 +321,7 @@ int set_cpu_coherent(void)
 
 int coherency_available(void)
 {
-	return coherency_type() != COHERENCY_FABRIC_TYPE_NONE;
+	return false;
 }
 
 int __init coherency_init(void)
-- 
2.7.4

